tests/integration/targets/acls_remediate/tasks/main.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Debug
+  ansible.builtin.debug:
+    msg: "START network.acls remediate role integration tests on connection={{ ansible_connection }}"
+
+- name: Cleanup, Set and remediate
+  block:
+    - name: Reset configuration
+      ansible.builtin.include_tasks: _cleanup.yaml
+
+    - name: Set resources
+      ansible.builtin.set_fact:
+        test_resources:
+          - "acls"
+          - "acl_interfaces"
+
+    - name: Set local data_store
+      ansible.builtin.set_fact:
+        test_data_store:
+          local: "~/test_backup"
+
+    - name: Run network.acls validated content with persist operation
+      ansible.builtin.include_role:
+        name: network.acls.persist
+      vars:
+        data_store: "{{ test_data_store }}"
+
+    - name: Assert file generations
+      ansible.builtin.include_tasks: includes/assertions.yaml
+      loop: "{{ test_resources }}"
+      loop_control:
+        loop_var: filename
+
+    - name: Run network.acls validated content with remediate operation
+      ansible.builtin.include_role:
+        name: network.acls.remediate
+      vars:
+        operations:
```

Comment:
I believe we don;t need this anymore

Reply:
Ah yes right The operations variable is no longer needed as we are now focusing on dedicated roles for specific tasks